core/abbreviate/abbreviate.py

```diff
@@ -72,22 +82,26 @@
     "document": "doc",
     "documents": "docs",
     "double": "dbl",
+    "double ended queue": "deque",
     "dupe": "dup",
     "duplicate": "dup",
     "dynamic": "dyn",
     "encode": "enc",
     "entry": "ent",
     "enumerate": "enum",
     "environment": "env",
+    "error": "err",
     "escape": "esc",
     "etcetera": "etc",
     "example": "ex",
     "exception": "exc",
     "execute": "exec",
-    "expression": "exp",
+    "exponent": "exp",
+    "expression": "expr",
```

Comment:
This is technically a breaking change, but I much more commonly see `expr` as an abbreviation for "expression", presumably thanks to [`expr(1)`].

[`expr(1)`]: https://en.wikipedia.org/wiki/expr